go.mod

```diff
@@ -5,9 +5,9 @@ go 1.18
 require (
 	github.com/IBM-Cloud/power-go-client v1.5.4
 	github.com/IBM/go-sdk-core/v5 v5.15.0
-	github.com/IBM/platform-services-go-sdk v0.54.0
+	github.com/IBM/platform-services-go-sdk v0.54.1
 	github.com/hashicorp/hcl/v2 v2.19.1
-	github.com/hashicorp/packer-plugin-sdk v0.4.0
```

Comment:
@Prajyot-Parab can you check Hashicorp's recent licensing thing effecting this module and the version updated here?

Reply:
will check on this.